mixins/bits/templates/bits-service-account.yml

```diff
@@ -6,6 +6,12 @@ kind: ServiceAccount
 metadata:
   name: bits-service
   namespace: {{ .Release.Namespace }}
+{{- if gt (len .Values.kube.image_pull_secrets) 0 }}
+imagePullSecrets:
+  {{- range $secret_name := $.Values.kube.image_pull_secrets }}
+  - name: "{{ $secret_name }}"
```

Comment:
It doesn't matter here because kube object names cannot have embedded double-quotes, but I feel `{{ $secret_name | quote }}` is idiomatic because it will be correct even when the variable value includes quotes itself.

Also add `| uniq` in the `range` expression above.